src/AddIns/Uno.UI.Lottie/LottieVisualSource.Android.cs

```diff
@@ -29,9 +66,13 @@ public void Update(AnimatedVisualPlayer player)
 		{
 			if (_animation == null)
 			{
+				_listener = new LottieListener(this);
+
 				_animation = new LottieAnimationView(Android.App.Application.Context);
 				_animation.EnableMergePathsForKitKatAndAbove(true);
-				_animation.UseHardwareAcceleration(UseHardwareAcceleration);
+				//_animation.UseHardwareAcceleration(UseHardwareAcceleration);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this commented out code.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5436651)

Reply:
+1